src/js/tech/html5.js

```diff
@@ -96,24 +115,28 @@ class Html5 extends Tech {
    * @method dispose
    */
   dispose() {
-    let tt = this.el().textTracks;
-    let emulatedTt = this.textTracks();
-
-    // remove native event listeners
-    if (tt && tt.removeEventListener) {
-      tt.removeEventListener('change', this.handleTextTrackChange_);
-      tt.removeEventListener('addtrack', this.handleTextTrackAdd_);
-      tt.removeEventListener('removetrack', this.handleTextTrackRemove_);
-    }
-
-    // clearout the emulated text track list.
-    let i = emulatedTt.length;
+    let trackTypes = ['audio', 'video', 'text'];
+
+    // ProxyNativeTextTracks
+    for (let i = 0; i < trackTypes.length; i++) {
+      let type = trackTypes[i];
+      let capitalType = type.charAt(0).toUpperCase() + type.slice(1);
+      let tl = this.el_[`${type}Tracks`];
+      let etl = this[`${type}Tracks`]();
+
+      if (tl && tl.removeEventListener) {
+        tl.removeEventListener('change', this[`handle${capitalType}TrackChange_`]);
```

Comment:
I think these tests should cover that https://github.com/BrandonOCasey/video.js/blob/integrate-video-audio-track/test/unit/tech/html5.test.js#L253